### PR TITLE
Publish updates from master

### PR DESCRIPTION
**Pull request by usha-mandya**
Publish the latest changes from master

### CODE REVIEW

**Comment by usha-mandya** [comment]
@thaJeztah @StefanScherer PTAL

File: compose/reference/up.md [hunk 1]
```diff
@@ -44,8 +45,8 @@ Builds, (re)creates, starts, and attaches to containers for a service.
 
 Unless they are already running, this command also starts any linked services.
 
-The `docker-compose up` command aggregates the output of each container (essentially running `docker-compose logs -f`). When
-the command exits, all containers are stopped. Running `docker-compose up -d`
+The `docker-compose up` command aggregates the output of each container (essentially running `docker-compose logs --follow`). When
```
Comment (thaJeztah) [inline compose/reference/up.md:48]: Do we know if these changes were also made upstream (or are these parts not in the cli help output of compose?)

**Review by thaJeztah — APPROVED** [review]
LGTM

File: compose/reference/up.md (hunk 1 above)
Comment (StefanScherer) [reply compose/reference/up.md:48]: No, there was only a PR to change it here. The usage of docker-compose shows both the short and long options.

**Review by binman-docker — APPROVED** [review]

**Review by shawnaxsom — APPROVED** [review]

**Pull request merged**